cypress/integration/landing.js

```diff
@@ -7,16 +7,40 @@ const selectors = {
   landingPageImage: '.landing-page-image'
 };
 
-describe('Landing page', function() {
-  it('renders', function() {
+describe('Landing page', () => {
+  it('Should render', () => {
     cy.visit('/');
     cy.title().should('eq', 'Learn to code at home | freeCodeCamp.org');
-    cy.contains(selectors.heading, 'Learn to code at home.');
     cy.contains(selectors.callToAction, "Get started (it's free)");
     cy.get(selectors.callToAction).should('have.length', 2);
   });
 
-  it('has a visible large image on large viewports', function() {
+  it('Has visible header and sub-header', () => {
+    cy.contains(selectors.heading, 'Learn to code at home.');
+    cy.contains('Build projects.').should('be.visible');
+    cy.contains('Earn certifications.').should('be.visible');
+
+    cy.contains(
+      'Since 2014, more than 40,000 freeCodeCamp.org ' +
+        'graduates have gotten jobs at tech companies including:'
+    ).should('be.visible');
+  });
+
+  it('Has 5 brand logos', () => {
+    cy.get('.logo-row')
+      .children()
+      .its('length')
+      .should('eq', 5);
+  });
+
+  it('Has testimonial section', () => {
```

Comment:
It is not clear how it is testing the testimonials section.

Reply:
Ooops! Sorry, my bad. The test name was really ambiguous. I've updated the test name now. Let me know what you think.